native/src/components/__tests__/MaterialHeaderButtons.spec.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import { render } from '@testing-library/react-native'
+import MaterialHeaderButtons from '../MaterialHeaderButtons'
+import buildConfig from '../../constants/buildConfig'
+import { Item } from 'react-navigation-header-buttons'
+
+describe('Material Header Buttons', () => {
```

Comment:
Hmm so these tests are kind of redundant as you are just asserting that the items you are specifying in the test have the opacity you give them. Instead it would be better to add these tests to the header tests and just render the `Header` component with different props, then you can assert that the opacity of the item with a specific accessibility label is either 0 or 1.

Reply:
i added some proper test for the header component :)